infra/staging/main.tf

```diff
@@ -32,6 +32,11 @@ variable "db_password" {
   sensitive = true
 }
 
+variable "frontend_bucket_name" {
+  type = string
+  default = "call-home-frontend"
```

Comment:
There is a good chance that we will have to rename this to something like `call-home-frontend-staging`, since we'll have a prod instance too, but we can cross that bridge when we get there.